Dockerfile

```
@@ -0,0 +1,7 @@
+FROM python:3
+ENV PYTHONUNBUFFERED 1
+RUN apt-get update --fix-missing && apt-get install -y python-dev libldap2-dev libsasl2-dev libssl-dev gettext libgettextpo-dev
+RUN mkdir /web
+WORKDIR /web
+COPY requirements.txt /web/
+RUN pip install -r requirements.txt
```

I think it's good practice to update `pip` before installing anything - which is what we do in the CI file ([`python.yml`](https://github.com/MAKENTNU/web/blob/b886772aaa580b8f7c2b397b60086c0be2ec131d/.github/workflows/python.yml#L32)) as well 😊 Though I'm unsure if `python` will use the intended Python version, or if we have to write `python3` or `python310` 🤔
```suggestion
COPY requirements.txt /web/
RUN python -m pip install --upgrade pip
RUN pip install -r requirements.txt
```

---

Tried to ouput the version numbers for different python versions, and can confirm that it does indeed use the intended version :)